### PR TITLE
UX improvement changes in namespace admin page

### PR DESCRIPTION
**Pull request by radhikav1**
# UX improvement changes in namespace admin page with mui components

## Description
New UX design changes in namespace admin page using MUI library components.
* Showing metrics using Card component.
* Old tabview is replaced with MUI Tabs component. 

## PR Type
- [ ] Bug Fix
- [x] Feature
- [ ] Build Fix
- [ ] Testing
- [x] General Improvement
- [ ] Cherry Pick

## Links
Jira: [CDAP-20741](https://cdap.atlassian.net/browse/CDAP-20741)

## Test Plan
Will be covered with e2e once complete feature is done.

## Screenshots
Proposed UX :
image
Implemented UI screen-shot:
image

[CDAP-20741]: https://cdap.atlassian.net/browse/CDAP-20741?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/cdap/components/NamespaceAdmin/AdminTabs.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
```
Comment (sumengwang) [inline app/cdap/components/NamespaceAdmin/AdminTabs.tsx]: year

File: app/cdap/components/NamespaceAdmin/Metrics/index.tsx [hunk 2]
```diff
@@ -60,21 +41,17 @@ const MetricsView: React.FC<IMetricsProps> = ({
...
-  const classes = useStyle();
-
   return (
-    <div className={classes.root}>
-      <div className={classes.content}>
-        <div className={classes.entitySection}>
-          <MetricColumn title="pipelines" metric={pipelinesCount} />
-          <MetricColumn title="datasets" metric={datasetsCount} />
-        </div>
+    <Box pt={2} pb={2}>
```
Comment (sumengwang) [inline app/cdap/components/NamespaceAdmin/Metrics/index.tsx]: make it into styled component

Comment (radhikav1) [reply app/cdap/components/NamespaceAdmin/Metrics/index.tsx]: Removed redundant containers.

File: app/cdap/components/NamespaceAdmin/AdminTabs.tsx (hunk 1 above)
Comment (radhikav1) [reply app/cdap/components/NamespaceAdmin/AdminTabs.tsx]: changed

File: app/cdap/components/NamespaceAdmin/AdminTabs.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+import { SourceControlManagement } from './SourceControlManagement';
+import { useFeatureFlagDefaultFalse } from 'services/react/customHooks/useFeatureFlag';
+import Box from '@material-ui/core/Box';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import TabContext from '@material-ui/lab/TabContext';
+import { styled } from '@material-ui/core/styles';
+import { useLocation } from 'react-router';
+
+const StyledTabs = styled(Tabs)({
```
Comment (sumengwang) [inline app/cdap/components/NamespaceAdmin/AdminTabs.tsx]: Is there a reason you are not using 
```
styled(Tabs)`something`
```
syntax?

**Review by sumengwang — APPROVED** [review]

File: app/cdap/components/NamespaceAdmin/AdminTabs.tsx (hunk 3 above)
Comment (radhikav1) [reply app/cdap/components/NamespaceAdmin/AdminTabs.tsx]: Nothing specific. I noticed that everywhere else the syntax is as you mentioned. I will change these as well to be consistent with other parts of code.

**Comment by njbriggs** [comment]
Please add a Jira for this work

**Review by GnsP — APPROVED** [review]

**Pull request merged**